### PR TITLE
wuzz: init at 2017-02-05

### PR DESCRIPTION
**Pull request by pradeepchhetri**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/networking/wuzz/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, pkgconfig, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "wuzz";
+  version = "89bebba7a4a00c52e97d411a3e82660189adc4ff";
```
Comment (joachifm) [inline pkgs/tools/networking/wuzz/default.nix]: Git revisions are invalid versions; please use the commit date in ISO 8601 format.

File: pkgs/tools/networking/wuzz/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, pkgconfig, buildGoPackage, fetchFromGitHub }:
```
Comment (joachifm) [inline pkgs/tools/networking/wuzz/default.nix]: `pkgconfig` is unused.

**Review by joachifm — CHANGES_REQUESTED** [review]

**Comment by pradeepchhetri** [comment]
@joachifm Thank you for the review. I have updated the pull request as per your suggestions.

File: pkgs/tools/networking/wuzz/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "wuzz";
```
Comment (joachifm) [inline pkgs/tools/networking/wuzz/default.nix]: The version must be embedded in the name, as in `wuzz-${version}`. The `version` attribute is not special; version strings are actually extracted from `name` using `parseDrvName`.

**Comment by pradeepchhetri** [comment]
@joachifm I messed up something while squashing multiple commits. Opening a new pull request https://github.com/NixOS/nixpkgs/pull/22495

**Comment by joachifm** [comment]
@pradeepchhetri you can also force-push the PR branch to fixup such issues

**Pull request closed without merging**